### PR TITLE
allow emails for calendar-maintainers group

### LINKED ISSUE #1046: allow anyone email calendar-maintainers@knative.team for one off-calendar events
On the [CALENDAR.MD](https://github.com/knative/community/blob/main/CALENDAR.MD) we state 
>If there's a one-off calendar invite needed, please send an email to calendar-maintainers@knative.team with the details of the event and we can create the event for you.

But someone reported that they tried to send an email and it bounced back.

We need to enable the google group with `WhoCanPostMessage: "ANYONE_CAN_POST"` in [groups.yaml](https://github.com/knative/community/blob/main/groups/groups.yaml#L61)

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #1046

### CODE REVIEW

**Comment by csantanapr** [comment]
/hold

I don't think this work in terms of permissions, since the calendar has this group as manager

**Comment by upodroid** [comment]
Hmm, i made it like that to block changes in the root folder, we can move this group under toc or steering folder.

**Review by evankanderson — COMMENTED** [review]
Can we include other groups in here (directly or transitively)?

It would be nice to be able to include `wg-leads`, `steering` and `toc` groups here, rather than repeating the names.

**Comment by evankanderson** [comment]
I think at least it makes sense for steering to be able to approve the `groups.yaml` -- it _has_ an audit trail in case something jinky happens, and they are org admins and sort of a backstop for the organization as a whole.

**Comment by upodroid** [comment]
You should be able to nest groups and modify OWNERS to allow steering to modify the root of the groups folder.

**Review by upodroid — APPROVED** [review]
LGTM

File: groups/groups.yaml [hunk 1]
```diff
@@ -57,14 +57,18 @@ groups:
...
     owners:
-      - omerbensaadon@knative.team
       - pmorie@knative.team
       - thisisnotapril@knative.team
+      - anassi@google.com
+      - csantanapr@knative.team
+      - csantana23@gmail.com
+      - lball@knative.team
       - vaikas@knative.team
-    members:
```
Comment (vaikas) [inline groups/groups.yaml:68]: Just curious why this removal?

Comment (csantanapr) [reply groups/groups.yaml:68]: Made everyone owner instead of members, I tried to mimic the other groups.yaml files

I'm either fine owners or members

Comment (vaikas) [reply groups/groups.yaml:68]: Maybe I'm reading the diff wrong, but it seems like abrennan has been now removed?

File: groups/groups.yaml [hunk 2]
```diff
@@ -57,14 +57,18 @@ groups:
...
-      Folks that can manage Knative calendar
+      Folks that can manage Knative calendar for one-off calendar invites
     settings:
       AllowExternalMembers: "false"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
     owners:
-      - omerbensaadon@knative.team
       - pmorie@knative.team
       - thisisnotapril@knative.team
+      - anassi@google.com
```
Comment (vaikas) [inline groups/groups.yaml:67]: nit, it would be nice to have this alphabetized :)

File: groups/groups.yaml [hunk 3]
```diff
@@ -57,14 +57,18 @@ groups:
   - email-id: calendar-maintainers@knative.team
     name: calendar-maintainers
     description: |-
-      Folks that can manage Knative calendar
+      Folks that can manage Knative calendar for one-off calendar invites
     settings:
       AllowExternalMembers: "false"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
```
Comment (evankanderson) [inline groups/groups.yaml:63]: Since this group is only intended for ACLs, can we change this to (effectively) NOBODY_CAN_POST? 😁

Comment (kvmware) [reply groups/groups.yaml:63]: https://knative.slack.com/archives/CU9DGL4FM/p1651791266269549?thread_ts=1651763947.024839&cid=CU9DGL4FM

**Comment by evankanderson** [comment]
Looks like this just needs a `/hold cancel`, but I'm not sure why @upodroid put the hold in place, so I'm just going to mention him. 😁

**Comment by upodroid** [comment]
/hold cancel

I thought there was something outstanding.

**Pull request merged**